packages/user-account/src/utils/validator/validatorHandler.js

```diff
@@ -0,0 +1,22 @@
+// const boom = require('@hapi/boom');
+
+// function validatorHandler(schema, property) {
+//   return (req, res, next) => {
+//     const data = req[property]
+//     const { error } = schema.validate(data, { abortEarly: false });
+//     if (error) {
+//       next(boom.badRequest(error));
+//     }
+//     next();
+//   }
+// }
```

Comment:
We usually dont push comments to the repo, if is necessary add a comment indicating why is commented

Reply:
thanks for the comment, it stayed whit me after resolving a doubt i had when implementing joi